parl/remote/client.py

```diff
@@ -170,7 +176,8 @@ def _reply_heartbeat(self):
                     remote_constants.HEARTBEAT_TAG,
                     to_byte(self.executable_path),
                     to_byte(str(self.actor_num)),
-                    to_byte(str(elapsed_time))
+                    to_byte(str(elapsed_time)),
+                    to_byte(str(self.log_monitor_url)),
```

Comment:
The `log_monitor_url` is regarded as part of the client status. In each heartbeat reply, the client status is returned to `Master._create_client_monitor`, in which the status is later forwarded to the cluster_monitor to update status of this client. The `ClusterMonitor.update_client_status()` rewrites all the information at each heartbeat. Otherwise we can use additional method to store the `log_monitor_url` for each client while the client is connected to the master, if needed.